manifests/native.json

```diff
@@ -248,6 +248,134 @@
       "mdnPath": "Global_Objects/Array/values",
       "category": "native"
     },
+    {
+      "type": "native",
+      "moduleName": "math.acosh",
+      "nodeVersion": "0.10.0",
+      "replacement": "Math.acosh",
+      "mdnPath": "Global_Objects/Math/acosh",
+      "category": "native"
+    },
+    {
+      "type": "native",
+      "moduleName": "math.asinh",
```

Comment:
this one also doesn't seem to be on npm

Reply:
The same, https://github.com/es-shims/Math.asinh